src/test/java/api/requests/scenarios/LoanDueDatesAfterRecallTests.java

```diff
@@ -57,6 +62,10 @@
 public class LoanDueDatesAfterRecallTests extends APITests {
```

Comment:
Are there other tests which check the due date after a recall that could be moved to this more specific class?

Reply:
I haven't analyzed it well, but it seems they cover all the cases defined in https://issues.folio.org/browse/CIRC-203 (the original requirements for the feature).